package.json

```diff
@@ -9,7 +9,7 @@
     "danger": "danger",
     "export-translations": "./scripts/translations/export.js",
     "import-translations": "./scripts/translations/import.js",
-    "svg2pdf": "./scripts/svg2pdf.js",
+    "svg-to-pdf": "./scripts/svg2pdf.js",
```

Comment:
@vanwagonet I think this is how you envisioned it? https://github.com/instructure/canvas-ios/blob/master/scripts/svg2pdf.js#L9

Reply:
oh, I originally had it as svg-to-pdf and renamed it and forgot to update the comment. lazy typing wanted no dashes.

Reply:
Okay I'll change the comment then. 😄